docs/use-mainnet/bridges-of-linea/index.mdx

```diff
@@ -232,3 +232,34 @@ Automatic claiming is only available for ETH L1 to L2 bridging. You will not be
 8. MetaMask will ask you to switch networks and to confirm the transaction from the previous step.
 
 9. You're done! You can check your Ethereum wallet balance on MetaMask or a block explorer such as [etherscan](https://etherscan.io/)!
+
+## How do I bridge ERC20 tokens?
+
+1. Go to the token selection menu.
```

Comment:
Can you add the link to where this UI is

Reply:
specified that it is the same token bridge. should be good now